### ISSUE TITLE
Warn on "else" usage when "if" branch ends in terminating statement

### ISSUE THREAD

**Issue by szepeviktor** [slevomat/coding-standard#495]
```php
if (condition) {
    ...
    break/continue/return/die/exit;
} else {
    echo 'Please do not use else!';
}
```

Thanks.

Was https://github.com/phpstan/phpstan/issues/1448

**Comment by kukulich** [comment]
The sniff already exists.

**Comment by szepeviktor** [comment]
@kukulich Thank you.

I've only found `SlevomatCodingStandard.ControlStructures.UselessConditionWithReturn`
which does not apply here.

Could you help me find that sniff?

**Comment by szepeviktor** [comment]
`SlevomatCodingStandard.ControlStructures.EarlyExit` is not triggered by the above code.

**Comment by kukulich** [comment]
Hmm, it looks you are right - different situation is reported.

**Comment by kukulich** [comment]
Implemented in https://github.com/slevomat/coding-standard/commit/54263691e348366f931129b19088be8b2e85b300

**Comment by szepeviktor** [comment]
Thank you.
